<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
background:none;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background:#222222;</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>501</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{border:none;
background:#FFF;
border-top-left-radius:2px;
border-bottom-left-radius:20px;
border-top-right-radius:20px;
border-bottom-right-radius:2px;
font-style:italic;
text-transform:capitalize;}
QListWidget::item{
padding-left:10px;
padding-bottom:3px;
 background:#FFF;
color:#000;
padding-top:4px; 
margin-top: 18px}
QListWidget::item:selected{color:#FFF;
background:#B8BEBF;
border:none;
border-top-left-radius:2px;
border-top-right-radius:8px;
border-bottom-left-radius:8px;
border-bottom-right-radius:2px;}
QListWidget::item:hover{
background:#A6DCE9;
border-top-left-radius:2px;
border-top-right-radius:8px;
border-bottom-left-radius:8px;
border-bottom-right-radius:2px;}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="descriprionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>491</width>
      <height>141</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>491</width>
      <height>141</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-family: &quot;Times New Roman&quot;;
font-size:20px;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#00FF6D;
font-family: &quot;Times New Roman&quot;;
font-size:24px;
font-weight:bold;
text-transform:uppercase;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="etudiantLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>431</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>431</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFF;
font-family: &quot;Times New Roman&quot;;
font-size:24px;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#FFF;
color:#B8BEBF;
border-radius:15px;
padding-left:13px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>   Recherche...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: arial;
color:#FFF;
text-decoration:none;</string>
    </property>
    <property name="text">
     <string>Par:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: arial;
color:#FFF;
text-decoration:none;</string>
    </property>
    <property name="text">
     <string>STATUT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-family: arial;
color:#FFF;
text-decoration:none;</string>
    </property>
    <property name="text">
     <string>Titre:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>537</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color : #B8BEBF;
                   color : #000000;
                   border : 2px solid #7A7D7E;
                   padding : 10px;
                   border-radius : 5px;
                   font-size : 16px;
                   }
QPushButton:hover{
                            background-color : #A3A8A9;
                            border : 2px solid #5C5F60;
                            }</string>
    </property>
    <property name="text">
     <string>Ajout de l'evaluation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>537</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color : #47BBC3;
                   color : #000000;
                   border : 2px solid #7A7D7E;
                   padding : 10px;
                   border-radius : 5px;
                   font-size : 16px;
                   }
QPushButton:hover{
                            background-color : #3DA8A0;
                            }
</string>
    </property>
    <property name="text">
     <string>Voir l'evaluation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: arial;
color:#FFF;
text-decoration:none;</string>
    </property>
    <property name="text">
     <string>Decription:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>481</width>
      <height>81</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>481</width>
      <height>81</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFF;
font-family: &quot;Times New Roman&quot;;
font-size:24px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey;
border-radius:7px;
background:#DCDCDC;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>299</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
padding-left:10px;
text-transform:uppercase;
font-family:Arial;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Evaluation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font-family:Arial;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>FeedBack:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>271</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>271</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0.5px solid grey;
background:#FFF;
color:#000;
font-family:Arial;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font-family:Arial;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>Note:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0.5px solid grey;
background:#FFF;
color:#000;
font-family:Arial;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid grey;
color:white;
background:#75BBE2;
font-weight:bold;</string>
      </property>
      <property name="text">
       <string>Mettre à jour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#E54259;
border:2px solid grey;
color:white;
font-weight:bold;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>label_6</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_7</zorder>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey;
border-radius:7px;
background:#DCDCDC;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>259</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#E54259;
border:2px solid grey;
color:white;
font-weight:bold</string>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font-family:Arial;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>311</width>
        <height>141</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>311</width>
        <height>141</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0.5px solid grey;
background:#FFF;
color:#000;
font-family:Arial;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Note:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>descriprionLabel</zorder>
   <zorder>listWidget</zorder>
   <zorder>statusLabel</zorder>
   <zorder>etudiantLabel</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
   <zorder>label_10</zorder>
   <zorder>scrollArea</zorder>
   <zorder>scrollArea_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
